### ISSUE TITLE
page parameter seems broken

### ISSUE THREAD

**Issue by cookie33** [EUDAT-B2HANDLE/B2HANDLE-HRLS#11]
Hi,

Testing with the latest devel branch shows following:
```
$  curl -u "user:password" "https://fqdn:8001/hrls/handles?URL=*&limit=10&page=0" |  python -m json.tool | sort
[
]
    "841/00c5a924-9cd4-11e5-8a71-040091643bd0", 
    "841/01184526-9cd4-11e5-8fea-040091643bd0", 
    "841/017e14b4-9cd4-11e5-8f00-040091643bd0", 
    "841/01cf5860-9cd4-11e5-ba16-040091643bd0", 
    "841/0230e558-9cd4-11e5-a1c6-040091643bd0", 
    "841/0243a886-7c60-11e4-a748-780dd91bf992", 
    "841/02617ca8-b250-11e6-acf4-5254000df0ed", 
    "841/02b39e64-f3d6-11e4-a518-5254000df0ed", 
    "841/04334470-9cd1-11e5-a431-040091643bd0", 
    "841/0481380a-f3d1-11e4-a4aa-5254000df0ed"

$  curl -u "user:password" "https://fqdn:8001/hrls/handles?URL=*&limit=10&page=1" |  python -m json.tool | sort
[
]
    "841/01184526-9cd4-11e5-8fea-040091643bd0", 
    "841/017e14b4-9cd4-11e5-8f00-040091643bd0", 
    "841/01cf5860-9cd4-11e5-ba16-040091643bd0", 
    "841/0230e558-9cd4-11e5-a1c6-040091643bd0", 
    "841/0243a886-7c60-11e4-a748-780dd91bf992", 
    "841/02617ca8-b250-11e6-acf4-5254000df0ed", 
    "841/02b39e64-f3d6-11e4-a518-5254000df0ed", 
    "841/04334470-9cd1-11e5-a431-040091643bd0", 
    "841/0481380a-f3d1-11e4-a4aa-5254000df0ed", 
    "841/0a2b604e-ab42-11e6-8687-04040a6400c1"

```
The servlet is build on centos7 and run on centos6.

It seems the limit and page do not work as expected. It is shifted 1 (one) instead of 10

And with an output file:
```
$  curl -u "user:password" "https://fqdn:8001/hrls/handles?URL=*&limit=10&page=0" |  python -m json.tool | sort > /tmp/out0.txt

$  curl -u "user:password" "https://fqdn:8001/hrls/handles?URL=*&limit=10&page=1" |  python -m json.tool | sort > /tmp/out1.txt

$ diff /tmp/out0.txt /tmp/out1.txt --side-by-side
[                                                               [
]                                                               ]
    "841/00c5a924-9cd4-11e5-8a71-040091643bd0",               <
    "841/01184526-9cd4-11e5-8fea-040091643bd0",                     "841/01184526-9cd4-11e5-8fea-040091643bd0", 
    "841/017e14b4-9cd4-11e5-8f00-040091643bd0",                     "841/017e14b4-9cd4-11e5-8f00-040091643bd0", 
    "841/01cf5860-9cd4-11e5-ba16-040091643bd0",                     "841/01cf5860-9cd4-11e5-ba16-040091643bd0", 
    "841/0230e558-9cd4-11e5-a1c6-040091643bd0",                     "841/0230e558-9cd4-11e5-a1c6-040091643bd0", 
    "841/0243a886-7c60-11e4-a748-780dd91bf992",                     "841/0243a886-7c60-11e4-a748-780dd91bf992", 
    "841/02617ca8-b250-11e6-acf4-5254000df0ed",                     "841/02617ca8-b250-11e6-acf4-5254000df0ed", 
    "841/02b39e64-f3d6-11e4-a518-5254000df0ed",                     "841/02b39e64-f3d6-11e4-a518-5254000df0ed", 
    "841/04334470-9cd1-11e5-a431-040091643bd0",                     "841/04334470-9cd1-11e5-a431-040091643bd0", 
    "841/0481380a-f3d1-11e4-a4aa-5254000df0ed"                |     "841/0481380a-f3d1-11e4-a4aa-5254000df0ed", 
                                                              >     "841/0a2b604e-ab42-11e6-8687-04040a6400c1"
```

So there might be a feature in the code..

**Comment by cookie33** [comment]
Tested with war file build on centos6. Same error. So war seems broken for page function.

**Comment by cookie33** [comment]
Fixed in #15
